javascript/javascript1/week4/homeWork/vowelCount.js

```diff
@@ -0,0 +1,18 @@
+const getWord = prompt("Enter any Word");
+function vowelCount(word) {
+  let count = 0;
+  const splittedWords = word.split("");
+  const vowelsList = "aeiouAEIOU".split("");
```

Comment:
This is smart adding the upper case letters in here. However, a more performant solution would be to just convert the function property to lower case earlier, so you don't need to loop extra times in the `if` statement.
```suggestion
  word = word.toLowerCase();
  const vowelsList = "aeiou".split("");
```

Reply:
Oh yes , now i got it ,

Reply:
Thank you again for your review :)